app/components/channel_bookmarks/channel_bookmark/bookmark_document.tsx

```diff
@@ -0,0 +1,76 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import React, {useCallback, useRef, useState} from 'react';
+import {StyleSheet, View} from 'react-native';
+import Button from 'react-native-button';
+
+import ProgressBar from '@app/components/progress_bar';
+import {useTheme} from '@app/context/theme';
+import Document, {type DocumentRef} from '@components/document';
+
+import BookmarkDetails from './bookmark_details';
+
+import type ChannelBookmarkModel from '@typings/database/models/servers/channel_bookmark';
+import type FileModel from '@typings/database/models/servers/file';
+
+type Props = {
+    bookmark: ChannelBookmarkModel;
+    canDownloadFiles: boolean;
+    file: FileModel;
+    onLongPress: () => void;
+}
+
+const styles = StyleSheet.create({
+    container: {
+        flexDirection: 'row',
+        paddingVertical: 6,
+        paddingLeft: 8,
```

Comment:
@enahum I think we can remove this left padding. Then we can adjust the spacing between bookmarks from 16 to 20. I'll comment on that line as well